File: ios/CounterManager.m
//
//  CounterManager.m
//  ViewCounterApp
//
//  Created by C84396A on 17/11/20.
//

#import "React/RCTViewManager.h"
@interface RCT_EXTERN_MODULE(CounterManager, RCTViewManager)
RCT_EXPORT_VIEW_PROPERTY(count, NSNumber)
@end

File: ios/CounterManager.swift
//
//  CounterManager.swift
//  ViewCounterApp
//
//  Created by C84396A on 17/11/20.
//

import Foundation

@objc(CounterManager)
class CounterManager : RCTViewManager {
  override func view() -> UIView! {
    return Counter()
  }
  
  override static func requiresMainQueueSetup() -> Bool {
    return true
  }
}
